pallets/nfts/src/benchmarking.rs

```diff
@@ -7,7 +7,7 @@ use sp_std::{boxed::Box, prelude::*};
 benchmarks! {
     create {
         let caller: T::AccountId = whitelisted_caller();
-    }: _(RawOrigin::Signed(caller), Default::default())
+    }: _(RawOrigin::Signed(caller), Default::default(), None)
```

Comment:
We should bench with some fake details instead of `None`

Reply:
That's true. If didn't want to fully fix the tests or the benchmark code until I got a approval that my approach makes sense :D